### DISCUSSION TITLE
Please let me know the metric I should use to find Page Load time

### DISCUSSION THREAD

**Discussion by Muthu-Krishnan-L** [GoogleChrome/lighthouse#15016] [Q&A]
I want to know which performance metric in Lighthouse can be used to identify the Full Page Load time of the webpage.

By Full Load Time I mean, when the entire page elements are loaded and ready for customer use. The load time that is equivalent to the load times mentioned in performance testing tools such as Jmeter. 

So far I was using TTI - but since it is removed now from Version 10 - please suggest

**Comment by LGNDDOLLABOUTIQUE** [comment]
You can use the google Lighthouse app or you can do it online.

**Reply by Muthu-Krishnan-L** [reply]
Kindly requesting you to read my question again and provide answer. I am asking which performance metric can be considered as Page Load time

**Comment by adamraine** [comment]
The current best estimate of "ready for customer use" we have is [LCP](https://web.dev/lcp/) which is meant to capture when the *user* perceives the main content of the page as loaded.

If you still wish to use TTI, it's available in the result JSON as I mentioned in https://github.com/GoogleChrome/lighthouse/issues/15015.

**Comment by paulirish** [comment]
Google Analytics's "Page Load Time" was === `loadEventStart`  which is == `onload` but without the cost of any event handlers. (which is dumb). Regardless, onload is a bad metric.

Any tools still showing 'page load time' don't care communicating web performance details with any sort of accuracy.  

https://web.dev/user-centric-performance-metrics/
